apps/checkout/src/lib/utils/utils.ts

```diff
@@ -16,13 +16,16 @@ export const getDataWithToken = <TData extends {} = {}>(
 });
 
 export type QueryVariables = Record<
-  "checkoutToken" | "passwordResetToken" | "email",
+  "checkoutToken" | "passwordResetToken" | "email" | "orderToken",
   string
 >;
 
 export const getQueryVariables = (): Partial<QueryVariables> => {
   const vars = queryString.parse(location.search);
-  return { ...vars, passwordResetToken: vars.token as string | undefined };
+  return {
+    ...vars,
+    passwordResetToken: vars.token as string | undefined,
```

Comment:
I think since order token is actually not required for checkout to work it should also be typed as `string | undefined` (and some other as well). We could make the type better:
```
export type QueryVariables = Record<
  "checkoutToken", string
> & Partial<Record<"orderToken" | "email" | "passwordResetToken">>;

Reply:
The type is already wrapped around a Partial `Partial<QueryVariables>`, so the type is `string | undefined`